app/src/main/cpp/skyline/nce/guest.h

```diff
@@ -0,0 +1,20 @@
+#pragma once
+
+namespace skyline {
+    namespace guest {
```

Comment:
We should add a message to be wary of what functions are used in guest.cpp

Reply:
Will do.